pkg/go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/djherbis/times v1.2.0
 	github.com/docker/docker v0.0.0-20170504205632-89658bed64c2
 	github.com/dustin/go-humanize v1.0.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
@EvanBoyle, are we switching away from glog to klog? I've lost track.

Reply:
We're sticking with glog right now. The klog change has been reverted.